.changeset/tidy-boxes-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search': patch
```

Comment:
I think it would be minor because we will introduce a breaking change. Previously the useSearch hook exported from the Search Plugin returned a toggleModal method and an open property, now it will return undefined for both, causing an error for those who used them.

Reply:
true, I missed that. Thanks!